src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/LoggingSourceConfiguration.cs

```diff
@@ -11,26 +12,33 @@ namespace Microsoft.Diagnostics.Monitoring.EventPipe
 {
     public class LoggingSourceConfiguration : MonitoringSourceConfiguration
     {
+        private const string UseAppFilters = "UseAppFilters";
+
         private readonly LogLevel _level;
+        private readonly bool _useAppFilter;
 
-        public LoggingSourceConfiguration(LogLevel level = LogLevel.Debug)
+        public LoggingSourceConfiguration(LogLevel level = LogLevel.Debug, bool useAppFilter = false)
```

Comment:
Please add an XML comment that this class will use the application filters instead of setting the log level for all enabled categories if userAppFilters = true.